save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/DeleteOrganizationButton.kt

```diff
@@ -0,0 +1,107 @@
+/**
+ * Utilities for cli args parsing
+ */
+
+@file:Suppress("FILE_NAME_MATCH_CLASS")
+
+package com.saveourtool.save.frontend.components.views.usersettings
+
+import com.saveourtool.save.frontend.components.modal.displayModal
+import com.saveourtool.save.frontend.utils.*
+import com.saveourtool.save.frontend.utils.noopLoadingHandler
+import csstype.ClassName
+import react.ChildrenBuilder
+import react.FC
+import react.Props
+import react.dom.html.ReactHTML.button
+import react.dom.html.ReactHTML.div
+import react.useState
+
+/**
+ * Button for delete organization
+ *
+ * @return noting
+ */
+val deleteOrganizationButton: FC<DeleteOrganizationButtonProps> = FC { props ->
+    val windowOpenness = useWindowOpenness()
+    val errorWindowOpenness = useWindowOpenness()
+    val (errorMessage, putErrorMessage) = useState<String?>(null)
+
+    displayModal(
```

Comment:
Suggest to use only one `displayModal` per page. You should mix this one with the one below.

Reply:
Done